test/index.js

```diff
@@ -157,63 +149,58 @@ test ('split runner syntax', done => {
            lib => { eq (lib, right); }],
   };
 
-  const run = createRunner (left, right, {}, spec);
-
-  run (options);
+  return executeRunner (createRunner (left, right, {}, spec), {
+    colors: false,
+    config: {minSamples: 1},
+    significantDifference: 0,
+  });
 });
 
-test ('no matches', () => {
-  const options = {
-    match: 'this will match nothing',
-  };
+await test ('no matches', async () => expectOutput (
+  [/^No benchmarks matched$/, /^$/],
+  () => {
+    const options = {
+      match: 'this will match nothing',
+    };
 
-  const spec = {
-    'this will not be matched': [{}, () => {}],
-  };
+    const spec = {
+      'this will not be matched': [{}, () => {}],
+    };
 
-  const run = createRunner ({}, {}, {}, spec);
-  const assert = expectOutput ([/^No benchmarks matched$/, /^$/]);
+    const run = createRunner ({}, {}, {}, spec);
 
-  run (options);
-  assert ();
-});
+    return executeRunner (run, options);
+  }
+));
 
-test ('no significant difference', done => {
-  const spec = {mock: [{}, () => {}]};
-  const run = createRunner ({}, {}, {}, spec);
-  const assert = expectOutput ([
+await test ('no significant difference', () => expectOutput (
+  [
     /^# 1[/]1: mock *\r$/,
     /^┌[─┬]+┐$/,
     /^│ suite +│ left +│ right +│ diff +│ change +│ α │$/,
     /^├[─┼]+┤$/,
+    // eslint-disable-next-line max-len
```

Comment:
Instead, let's set `"max-len": ["off"]` for `test/**/*.js` in __.eslintrc.json__ as we do in other projects.